var/spack/repos/builtin/packages/cbtf-krell/package.py

```diff
@@ -104,6 +100,8 @@ class CbtfKrell(Package):
 
```

Comment:
Interesting, I've never heard of a package that could build with multiple MPI libraries. In that case, I guess it is fine as is.